deployments/charts/kuma/README.md

```diff
@@ -69,6 +69,8 @@ A Helm chart for the Kuma Control Plane
 | ingress.annotations | object | `{}` | Additional deployment annotation |
 | ingress.nodeSelector | object | `{"kubernetes.io/arch":"amd64","kubernetes.io/os":"linux"}` | Node Selector for the Ingress pods |
 | kumactl.image.repository | string | `"kumactl"` | The kumactl image repository |
+| kubectl.image.registry | string | `"bitnami"` | The kubectl image registry |
+| kubectl.image.repository | string | `"kubectl"` | The kubectl image repository |
 | hooks.nodeSelector | object | `{"kubernetes.io/arch":"amd64","kubernetes.io/os":"linux"}` | Node selector for the HELM hooks |
```

Comment:
Should't we add option to specify version/tag here?

Reply:
This one is autogenerated and unfortunatelly once we comment out the value it just cleans it during `make check`.

Reply:
what if we have empty value in `values.yaml` ?